crates/env/src/call/call_builder.rs

```diff
@@ -24,7 +24,7 @@ use crate::{
         ExecutionInput,
     },
     Environment,
-    Error,
```

Comment:
Not sure why any of the call builders are required in any of the environment implementations since they are a complete `ink_env` utility.